CHT/HeatTransferCoefficient.C

```diff
@@ -25,7 +25,7 @@ mesh_(mesh)
 }
 
 
-void preciceAdapter::CHT::HeatTransferCoefficient::write(double * buffer)
+void preciceAdapter::CHT::HeatTransferCoefficient::write(double * buffer, bool provideMeshConnectivity)
 {
```

Comment:
Why no implementation here? Don't we need at least a warning?

Reply:
Done